src/indigo/less/tables.less

```diff
@@ -1,5 +1,30 @@
 .table {
   thead tr {
     box-shadow: @box-shadow;
+
+    th {
+      border-bottom: none;
+    }
+  }
+
+  > thead,
+  > tbody,
+  > tfoot {
+    > tr {
+      > th,
+      > td {
+        line-height: 1.28571429;
```

Comment:
@natocTo pls, odkial je tato hodnota  a preco je to takto? :)

Reply:
Ani přesně nevím proč je to takto číslo, asi že to tak bylo i předtím tak jsme automaticky to upravil jen na jiné číslo. Jinak je to 18 / 14 (18px je line-height ve firmě pro tabulku a 14px tam je font size).

Reply:
ok, presun to potom v dalsom PR, do nejake premennej a tu nech sa to vyrata. ako napr.:

```less
line-height: @table-cell-height / @font-size-base;
```
... alebo nieco podobne

Reply:
okei